source/_components/switch.telnet.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Telnet Switch"
+description: "Instructions how to integrate telnet switches into Home Assistant."
+date: 2017-08-10 19:19
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Switch
+ha_release: 0.51
+ha_iot_class: "Local Polling"
+---
+
+
+The `telnet` switch platform allows you to control devices with telnet commands. It connects to a host on port 23.
+
+To enable this switch, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  platform: telnet
+  switches:
+    projector:
+      resource: "host_or_ip"
+      command_on: "PWR ON"
+      command_off: "PWR OFF"
+      command_state: "PWR?"
+      value_template: '{{ value == "PWR=01" }}'
+```
+
+Configuration variables:
+
+- **switches** (*Required*): The array that contains all switches.
+  - **identifier** (*Required*): Name of the switch as slug. Multiple entries are possible.
```

Comment:
I think this should be rephrased since the word "slug" isn't a commonly used phrase for users new to this area.